src/js/edu-benefits/5490/config/form.js

```diff
@@ -73,7 +75,14 @@ const formConfig = {
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
-      pages: {}
+      pages: {
+        applicantInformation: applicantInformation([
+          'relativeFullName',
```

Comment:
Are these the right fields to use? There are also veteranFullName, veteranSocialSecurityNumber, and veteranDateOfBirth.